### PR TITLE
fix dropdown not closing

### PR DESCRIPTION
**Pull request by noodlemoodle**

### CODE REVIEW

File: src/foam/u2/view/OverlayActionListView.js [hunk 1]
```diff
@@ -231,9 +231,14 @@ foam.CLASS({
 
       this.onDetach(() => { this.overlay_ && this.overlay_.remove(); });
 
+      // sub to actions from view
       self.obj?.sub(function() {
         self.overlay_.close();
       });
+      // sub to actions from view data
+      self.obj?.data?.sub(function() {
```
Comment (sarthak-marwaha) [inline src/foam/u2/view/OverlayActionListView.js]: Shouldnt this sub to the slot?

Comment (noodlemoodle) [reply src/foam/u2/view/OverlayActionListView.js]: the slot sub doesn't work, what's the reason it should sub to the slot instead?

Comment (sarthak-marwaha) [reply src/foam/u2/view/OverlayActionListView.js]: The object doesnt change, the slot is what would change, what is this trying to fix?

Comment (noodlemoodle) [reply src/foam/u2/view/OverlayActionListView.js]: we don't need to worry about the obj changing, we just need to know when an action on the object has been called

Comment (noodlemoodle) [reply src/foam/u2/view/OverlayActionListView.js]: this is to fix actions from the obj on the overlay not triggering the overlay close, made one more update

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**